[DCOS_OSS-5951] edits to quickstart (#2878)
mesosphere/dcos-docs-site: pages/ksphere/kommander/latest/quick-start/index.md

Download the tarball to your local Downloads directory.

For example, if you are installing on MacOS, download the compressed archive to the default `~/Downloads` directory.

> Afterward, extract the tarball to your local system by running the following command (make sure to double check the name of the file that you downloaded and change konvoy-kommander_darwin.tar.bz2 accordingly):

Reviewer: ```suggestion
Afterward, extract the tarball to your local system by running the following command (ensure the name of the file that you downloaded, and change konvoy-kommander_darwin.tar.bz2 accordingly):
```

```
tar -xf ~/Downloads/konvoy-kommander_darwin.tar.bz2
```

Copy the Konvoy package files to a directory in your user's `PATH` to ensure you can invoke the konvoy command from any directory.

For example, copy the package to `/usr/local/bin/` by returning to the directory that Kommander was extracted to, and running the following command: 

```
sudo cp ~/Downloads/darwin/konvoy-kommander.tar.bz2/* /usr/local/bin/
```

> NOTE: The extracted folder may have a different name (such as `konvoy_dev`, or perhaps it was given a custom directory, or have the version affixed to the end of the directory name)

Reviewer: ```suggestion
<p class="message--note"><strong>NOTE: </strong>The extracted folder may have a different name (such as `konvoy_dev`, or perhaps it was given a custom directory, or have the version affixed to the end of the directory name)</p>
```

### Logging in with Username and Password

> After you provision your first Konvoy cluster, your randomly-generated username, password, and a URL to Konvoy will be printed to the command-line. Once in Konvoy, you should see a button labelled "Try Kommander". If not, check to ensure you've installed the Konvoy release that includes Kommander.

Reviewer: ```suggestion
After you provision your first Konvoy cluster, your randomly-generated username, password, and a URL to Konvoy are printed to the command-line. Once in Konvoy, you should see a button labelled "Try Kommander". If not, ensure you've installed the Konvoy release that includes Kommander.
```
